server/src/utils.js

```diff
@@ -0,0 +1,6 @@
+const sleep = time => new Promise((resolve) => {
+  setTimeout(() => resolve(), time);
+});
+
+
+export default sleep;
```

Comment:
this will definitely have more things in it at some point:
```
export default { sleep };
```
import with:
```
import { sleep } from './utils';
```

Reply:
results in undefined function errors

Reply:
oh I know why.. you'd need to import it as:
```
import utils from '../utils';
```
and use:
```
await utils.sleep(1000);
```
but that's not good for utils so this is fine for now.